base_layer/wallet_ffi/.venv/lib/python3.12/site-packages/_pytest/doctest.py

```
@@ -0,0 +1,754 @@
+    real_unwrap = inspect.unwrap
+
+    def _mock_aware_unwrap(
+        func: Callable[..., Any], *, stop: Callable[[Any], Any] | None = None
+    ) -> Any:
+        try:
+            if stop is None or stop is _is_mocked:
+                return real_unwrap(func, stop=_is_mocked)
+            _stop = stop
+            return real_unwrap(func, stop=lambda obj: _is_mocked(obj) or _stop(func))
+        except Exception as e:
+            warnings.warn(
```

Ruff (0.11.9)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)